src/View/Form/EntityContext.php

```diff
@@ -534,10 +547,10 @@ protected function _getValidator(array $parts): Validator
     /**
      * Get the table instance from a property path
      *
-     * @param array|string|\Cake\Datasource\EntityInterface $parts Each one of the parts in a path for a field name
+     * @param string[]|string|\Cake\Datasource\EntityInterface $parts Each one of the parts in a path for a field name
      * @param bool $fallback Whether or not to fallback to the last found table
      *  when a non-existent field/property is being encountered.
-     * @return \Cake\ORM\Table|bool Table instance or false
+     * @return \Cake\ORM\Table|false Table instance or false
```

Comment:
How about making it return `null` instead of `false` if no matching table is found?

Reply:
I agree with you. But can we still do that here in 4.x?

Reply:
Sure can.